src/main/java/com/github/hanyaeger/knightlyperil/entities/characters/enemy/IEnemy.java

```diff
@@ -0,0 +1,10 @@
+package com.github.hanyaeger.knightlyperil.entities.characters.enemy;
+
+import com.github.hanyaeger.api.entities.impl.DynamicSpriteEntity;
+
+/**
+ * An IEnemy represents an Enemy. This interface is used to distinguish between types of colliders.
+ */
+public interface IEnemy {
+
```

Comment:
klopt het dat dit bestand leeg is?

Reply:
Deze interface is nodig voor het onderscheiden van de enemies.